src/main/java/io/personium/client/ServiceCollection.java

```diff
@@ -86,6 +107,60 @@ public void configure(String key, String value, String subject) throws DaoExcept
         rest.setService(this.getPath(), key, value, subject);
     }
 
+    /**
+     * This method configures a set of services.
+     * @param routes list of route
+     * @param subject Value of the service subject
+     * @throws DaoException Exception thrown
+     */
+    public void configure(List<IPersoniumServiceRoute> routes, String subject) throws DaoException {
```

Comment:
It is an overloaded method. Please adjust it to other configure method. For example, 

```java
RestAdapter rest = (RestAdapter) RestAdapterFactory.create(this.accessor);
rest.setService(this.getPath(), routes, subject);
```

Reply:
I think it is better like the following.

```java
public void configure(String key, String value, String subject)
    RestAdapter rest = (RestAdapter) RestAdapterFactory.create(this.accessor);
    String xml = createXMLString(key, value);
    rest.proppatch(this.getPath(), sw.toString());
}

public void configure(List<IPersoniumServiceRoute> routes, String subject) {
    RestAdapter rest = (RestAdapter) RestAdapterFactory.create(this.accessor);
    String xml = createXMLString(routes);
    rest.proppatch(this.getPath(), xml);
}
```